addon/components/perfect-scroll/component.js

```diff
@@ -44,18 +44,57 @@ export default Ember.Component.extend({
   useKeyboard: true,
   suppressScrollX: false,
   suppressScrollY: false,
-  scrollTop: 0,
-  scrollLeft: 0,
   scrollXMarginOffset: 0,
   scrollYMarginOffset: 0,
   includePadding: false,
   theme: 'default',
 
+  _scrollTop: 0,
```

Comment:
Yes, it is. All the tests written on theses computed properties are still passing.